lib/algos/list/delimiterMaster.js

```diff
@@ -165,14 +175,36 @@ class DelimiterMaster extends Delimiter {
         return super.filter(obj);
     }
 
+    /**
+     * Determine if a nextContinueMarker ending with a delimiter
+     * can be skipped.
+     * @param {String} key - a nextContinueMarker
+     * @returns {bool}
+     */
+
+    allowDelimiterRangeSkip(key) {
```

Comment:
I believe we don't really need to pass a parameter here, let's just use the member `this[this.nextContinueMarker]` in place (it's good to use a local variable like `currentMarker = this[this.nextContinueMarker]` though)